Sources/ApolloCodegenLib/FileGenerators/SchemaModuleFileGenerator.swift

```diff
@@ -30,6 +30,10 @@ struct SchemaModuleFileGenerator {
       return
     }
 
-    try fileManager.apollo.createFile(atPath: filePath, data: rendered.data(using: .utf8))
+    try fileManager.apollo.createFile(
+      atPath: filePath,
+      data: rendered.data(using: .utf8),
+      overwrite: true
```

Comment:
This file generator doesn't conform to `FileGenerator` so it has to pass in the `overwrite` value.

Reply:
We should make overwrite default to true at the function declaration.

Reply:
I debated that and then decided that function being generic to multiple file uses should require you to specify the value. I could be convinced otherwise though, `createFile` does imply action by default. Plus overwrite is the default behaviour of `FileManager`.

Reply:
OK..I just talked myself into it. It'll cut down on this diff size too.